src/test/scala/com/cognite/sdk/scala/v1/SecurityCategoriesTest.scala

```diff
@@ -0,0 +1,8 @@
+package com.cognite.sdk.scala.v1
+
+import com.cognite.sdk.scala.common.{ReadBehaviours, SdkTest}
+
+class SecurityCategoriesTest extends SdkTest with ReadBehaviours {
+  // We don't have any security categories in publicdata, and lack permissions to list them in greenfield
+  //"SecurityCategories" should behave like readable(client.securityCategories, supportsLimit = false)
```

Comment:
Ignore this test instead of commenting out

Reply:
Added @Ignore but it didn't work, so I left it in but commented out the whole thing instead.